images/linux/scripts/helpers/etc-environment.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+################################################################################
+##  File:  etc-environment.sh
+##  Desc:  Helper functions for source and modify /etc/environment
+################################################################################
+
+# NB: sed expression use '%' as a delimiter in order to simplify handling
+#     values containg slashes (i.e. directory path)
+#     The values containing '%' will break the functions
+
+function getEtcEnvironmentVar {
+    var_name="$1"
+    # remove `var_name=` and possible quotes from the line
+    grep "^${var_name}=" /etc/environment |sed -E "s%^${var_name}=\"?([^\"]+)\"?.*$%\1%"
+}
+
+function addEtcEnvironmentVar {
+    var_name="$1"
+    var_value="$2"
+
+    echo "$var_name=\"$var_value\"" | sudo tee -a /etc/environment
+}
+
+function replaceEtcEnvironmentVar {
+    var_name="$1"
+    var_value="$2"
+
+    # modify /etc/environemnt in place by replacing a string that begins with var_name
+    sudo sed -ie "s%^${var_name}=.*$%${var_name}=\"${var_value}\"%" /etc/environment
+}
+
+function setEtcEnvironmentVar {
```

Comment:
`setEtcEnvironmentVariable` - let's use the full `variable` word for readability
The same for other function

Reply:
The changes (and related to it) are implemented in the another PR: 
https://github.com/actions/virtual-environments/pull/660/commits/ba017c929014c482545acb2eb5a3fdaa4a71ddce
https://github.com/actions/virtual-environments/pull/660/commits/144559b5f252ca47b23345fcef5c27eff1b39537
https://github.com/actions/virtual-environments/pull/660/commits/a2517ced678009aba36207d0c61d740f7f5c35f0
https://github.com/actions/virtual-environments/pull/660/commits/3ebe6154b26c8807ab82b109bd3a85e0dc407b6a